### DISCUSSION TITLE
web端pagComposition使用问题

### DISCUSSION THREAD

**Discussion by zxcv66666** [Tencent/libpag#1730] [Q&A] [answered]
1.如何在不知道具体的height和width值的情况下，如何设置pagComposition能让他占满全屏，我看在make方法中好像必须传入height和width的值，而不是按照canvas的大小进行展示的，我给渲染的canvas设置全屏的大小 ，但是pagComposition渲染出来并不是全屏
2.pagComposition中的各个图层如何设置他的具体位置
感谢指导

**Comment by domchen** [comment, marked as answer]
1、必须知道宽高才行，你可以加个监听，等canvas的布局完成了读取一下宽高再创建PAGComposition。
2、通过PAGComposition.setLayerIndex()就行，具体可以读一下文档。这个图层树是一个紧凑的数组，移除后所有图层的索引会跟新。索引越小的在底部。
